### PR TITLE
[ODS-5497] - Include ApiClientId in WebApi log statements

### PR DESCRIPTION
**Pull request by bptillman**
Now for any log entries that relate to a specific ApiClient, the ApiClientId will now get logged as part of each entry.
You can see it in the diff view in the implementation repo PR, but the log entries will now look similar to this, where 2 is the ApiClientId on the first two entries since they relate to an ApiClient, while the third one does not have an ApiClientId since it is not related to an ApiClient:
```
2022-07-21 16:44:46,378 [.NET ThreadPool Worker,2] DEBUG AuthorizationBasisMetadataSelector - Authorization strategy 'NamespaceBased' selected for request against resource 'http://ed-fi.org/ods/identity/claims/ed-fi/assessment'.
2022-07-21 16:44:58,856 [.NET ThreadPool Worker,2] ERROR AssessmentsController - GetAllRequest
System.Exception: Exception of type 'System.Exception' was thrown.
2022-07-21 16:44:59,424 [.NET ThreadPool Worker,] DEBUG QuartzSchedulerThread - Batch acquisition of 0 triggers
```

### CODE REVIEW

**Review by AxelMarquez — CHANGES_REQUESTED** [review]
Overall looks good, just a few minor formatting changes

File: Artifacts/MsSql/Structure/Admin/0100-Add-ApiClient-to-GetClientForToken.sql [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+        ,ac.[UseSandbox]
+        ,ac.[StudentIdentificationSystemDescriptor]
+        ,aeo.[EducationOrganizationId]
+        ,app.[ClaimSetName]
+        ,vnp.[NamespacePrefix]
+        ,p.[ProfileName]
+        ,ac.[CreatorOwnershipTokenId_OwnershipTokenId] as CreatorOwnershipTokenId
+        ,acot.[OwnershipToken_OwnershipTokenId] as OwnershipTokenId
+        ,cat.Expiration
+		,cat.ApiClient_ApiClientId as ApiClientId
```
[the remark this answers is not in the document]
Comment (bptillman) [reply Artifacts/MsSql/Structure/Admin/0100-Add-ApiClient-to-GetClientForToken.sql]: This has been fixed.

File: Artifacts/PgSql/Structure/Admin/0100-Add-ApiclientId-to-GetClientForToken.sql [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+    ,UseSandbox BOOLEAN
+    ,StudentIdentificationSystemDescriptor VARCHAR(306)
+    ,EducationOrganizationId INT
+    ,ClaimSetName VARCHAR(255)
+    ,NamespacePrefix VARCHAR(255)
+    ,ProfileName VARCHAR
+    ,CreatorOwnershipTokenId SMALLINT
+    ,OwnershipTokenId SMALLINT
+    ,Expiration TIMESTAMP
+	,ApiClientId INT
```
[the remark this answers is not in the document]
Comment (bptillman) [reply Artifacts/PgSql/Structure/Admin/0100-Add-ApiclientId-to-GetClientForToken.sql]: This has been fixed.

File: Artifacts/PgSql/Structure/Admin/0100-Add-ApiclientId-to-GetClientForToken.sql [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+        ,ac.UseSandbox
+        ,ac.StudentIdentificationSystemDescriptor
+        ,aeo.EducationOrganizationId
+        ,app.ClaimSetName
+        ,vnp.NamespacePrefix
+        ,p.ProfileName
+        ,ac.CreatorOwnershipTokenId_OwnershipTokenId as CreatorOwnershipTokenId
+        ,acot.OwnershipToken_OwnershipTokenId as OwnershipTokenId
+        ,cat.Expiration
+		,cat.ApiClient_ApiClientId as ApiClientId
```
[the remark this answers is not in the document]
Comment (bptillman) [reply Artifacts/PgSql/Structure/Admin/0100-Add-ApiclientId-to-GetClientForToken.sql]: This has been fixed.

**Review by AxelMarquez — APPROVED** [review]

**Pull request merged**